### PR TITLE
Add Model Docstrings

### PR DESCRIPTION
**Pull request by ghickman**
This adds docstrings to each model to show their intended use.

### CODE REVIEW

File: jobserver/models.py [hunk 1]
```diff
@@ -602,6 +628,8 @@ def get_remove_url(self):
 
 
 class Release(models.Model):
+    """Outputs released from a Workspace"""
```
Comment (ghickman) [inline jobserver/models.py:631]: @bloodearnest – would love a quick check on this from you since you built the releases bits!

**Review by CarolineMorton — APPROVED** [review]

**Review by evansd — APPROVED** [review]

File: jobserver/models.py (hunk 1 above)
Comment (bloodearnest) [reply jobserver/models.py:631]: Maybe "Review and redacted outputs from a Workspace"?

Comment (ghickman) [reply jobserver/models.py:631]: Did you mean "Reviewed" here?

Comment (ghickman) [reply jobserver/models.py:631]: Assuming yes and updated commit with better words, thanks!

Comment (bloodearnest) [reply jobserver/models.py]: :facepalm:

**Pull request merged**